eng/common/scripts/get-codeowners.ps1

```diff
@@ -0,0 +1,43 @@
+param (
+  $TargetDirectory, # should be in relative form from root of repo. EG: sdk/servicebus
+  $RootDirectory # ideally $(Build.SourcesDirectory)
+)
+
+$codeOwnersLocation = @(Get-ChildItem -R -Force -Path $RootDirectory -Filter "CODEOWNERS")
+
+if ($codeOwnersLocation.Length -eq 0) {
+  Write-Host "Unable to find CODEOWNERS file in target directory $RootDirectory"
+  exit 1
+}
+
+if ($codeOwnersLocation.Length -gt 1) {
+  Write-Host "Multiple CODEOWNERS files detected in  $RootDirectory."
+  Write-Host "$codeOwnersLocation"
+  exit 1
+}
+
+
+$codeOwnersContent = Get-Content ($codeOwnersLocation | Select-Object -Last 1)
+
+$ownedFolders = @{}
+
+foreach ($contentLine in $codeOwnersContent) {
+  if (-not $contentLine.StartsWith("#") -and $contentLine){
+    $splitLine = $contentLine -split "\s" | ? { return $_ }
```

Comment:
Similar to your other usages I assume `? { return $_ }` is to try and filter out nulls? Does the split command even return nulls? If you do still need this for some reason you should be able to remove the return keyword as that is redundant.

Reply:
It's more than nulls. Also filters out `""`.

Reply:
Ok I think I understand why this is needed now and I agree as it does eliminate any empty matches.

` $splitLine = $contentLine -split "\s" | ? { $_ }`

Reply:
Made an adjustment to split regex to make it un-necessary.